docs/docs/building-applications/edge.md

```diff
@@ -1,10 +1,18 @@
 # Using Groundlight on the edge
 
-Starting your model evaluations at the edge reduces latency, cost, network bandwidth, and energy. Once you have downloaded and installed your Groundlight edge models, you can configure the Groundlight SDK to use your edge environment by configuring the 'endpoint' to point at your local environment as such:
+Starting your model evaluations at the edge reduces latency, cost, network bandwidth, and energy. Once you have downloaded and installed your Groundlight edge models, you can configure the Groundlight SDK to use your edge environment by configuring the 'endpoint' which the SDK connects to.  You can do this either directly in code as such:
 
 ```python
 from groundlight import Groundlight
 gl = Groundlight(endpoint="http://localhost:6717")
 ```
 
+or by setting the `GROUNDLIGHT_ENDPOINT` environment variable like:
+
+```bash
+export GROUNDLIGHT_ENDPOINT=http://localhost:6717
+python glapp.py
```

Comment:
took me a hot second to figure out glapp.py is just a generic name. Maybe your_app.py?

Reply:
Good idea.